custom_components/storj/backup.py

```diff
@@ -138,7 +142,9 @@ async def async_delete_backup(
 
             if backup:
                 await self._client.async_delete_backup(backup)
+                return
```

Comment:
`async_get_backup` now unconditionally returns a backup, so this can be simplified
```suggestion
            await self._client.async_delete_backup(backup)
```